pyaedt/hfss.py

```diff
@@ -1522,28 +1530,28 @@ def create_sbr_antenna(
             parameters_defaults["Array Weight Cosine Exp"] = 1
             parameters_defaults["Array Differential Pattern Type"] = 0
             if is_array:
-                antenna_name = generate_unique_name("pAntArray")
-        if parameters_dict:
-            for el, value in parameters_dict.items():
+                name = generate_unique_name("pAntArray")
+        if parameters:
+            for el, value in parameters.items():
                 parameters_defaults[el] = value
-        return self._create_native_component(antenna_type, target_cs, model_units, parameters_defaults, antenna_name)
+        return self._create_native_component(antenna_type, target_cs, units, parameters_defaults, name)
 
-    @pyaedt_function_handler()
+    @pyaedt_function_handler(ffd_full_path="far_field_data", model_units="units", antenna_name="name")
     def create_sbr_file_based_antenna(
         self,
-        ffd_full_path,
+        far_field_data,
         antenna_size="1mm",
         antenna_impedance="50ohm",
         representation_type="Far Field",
         target_cs=None,
-        model_units=None,
-        antenna_name=None,
+        units=None,
+        name=None,
     ):
         """Create a linked antennas.
```

Comment:
```suggestion
        """Create a linked antenna.
```
One linked attenna? Otherwise, "Create linked antennas."